app/src/main/java/com/example/githubusers/screens/main/StartViewModel.kt

```diff
@@ -17,9 +17,11 @@ class StartViewModel @Inject constructor(
 ) : ViewModelBase() {
 
     init {
-        viewModelScope.launch(Dispatchers.IO + coroutineExceptionHandler) {
-            repository.updateListApi()
-        }
+        updateUsersList()
+    }
+
+    fun updateUsersList() {
+        viewModelScope.launch(Dispatchers.IO) { repository.updateListApi() }
     }
```

Comment:
what if this method called several times until request is completed and new data is stored to DB?